src/main/drivers/timer_def.h

```diff
@@ -459,6 +459,11 @@
         DEF_TIM_DMA_STREAM(dmaopt, TCH_## tim ## _ ## chan),            \
         DEF_TIM_DMA_CHANNEL(dmaopt, TCH_## tim ## _ ## chan),           \
         DEF_TIM_DMA_HANDLER(dmaopt, TCH_## tim ## _ ## chan)            \
+    ),                                                                  \
+    DEF_TIM_DMA_COND(                                                   \
```

Comment:
Technically this should go to timer description, not timer channel. But it will work this way, maybe just comment it